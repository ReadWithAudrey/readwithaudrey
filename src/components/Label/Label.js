import React from 'react'
import { StyledLabel } from './Label.style'

const Label = () => <StyledLabel />

export default Label
