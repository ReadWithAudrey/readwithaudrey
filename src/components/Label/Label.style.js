import styled from 'styled-components'

export const StyledLabel = styled.label.attrs({
  className: 'white br2 ba bw1 b--white ',
})`
  color: var(--pink);
  display: block;
`
