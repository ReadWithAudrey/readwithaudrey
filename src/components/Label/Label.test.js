import React from 'react'
import renderer from 'react-test-renderer'
import Label from './Label'

describe('Label component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<Label />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
