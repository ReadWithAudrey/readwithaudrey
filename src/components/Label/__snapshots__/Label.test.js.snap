// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label component snapshot test renders correctly 1`] = `
<label
  className="sc-bdVaJa pink montserrat fw7 f6 pv2 fpcbkF"
/>
`;
